pypeit/scripts/collate_1d.py

```diff
@@ -437,25 +437,27 @@ def get_parser(cls, width=None):
                                                 'nights and prepare a directory for the KOA.',
                                     width=width, formatter=scriptbase.SmartFormatter)
 
+        # TODO: Is the file optional?  If so, shouldn't the first argument start
```

Comment:
A question for @badpandabear

Reply:
Yeah, it probably should, so that it's clear to users that it is optional and for consistency with pypeit_sensfunc. If you change that in this PR I'd appreciate a second short form "-i" for this option to save on typing as well. But If you prefer I can make this change in my next update pypeit_collate1d

Reply:
Yeah, I'll leave it to you to change.  I'll leave in the TODO comment for the time-being.  Thanks for having a look!